ui/cryptomaterial/editor.go

```diff
@@ -43,7 +50,10 @@ type Editor struct {
 	// isPassword if true, displays the show and hide button.
 	isPassword bool
 	// If showEditorIcon is true, displays the editor widget Icon of choice
-	showEditorIcon bool
+	showEditorIcon  bool
+	alignEditorIcon AlignmentIcon
```

Comment:
You can do without the `AlignmentIcon` type. By default, the icon is positioned to the left, right? So just add a `alignIconEnd bool` property here. It's false by default, so the icon position will be left/start by default.
```suggestion
	alignEditorIconEnd bool
```